commands/pull.js

```diff
@@ -30,6 +31,20 @@ function exec (cmd, opts = {}) {
   }
 }
 
+function spawn (cmd) {
+  const {spawn} = require('child_process')
+  return new Promise((resolve, reject) => {
+    let result = ''
+    let psql = spawn(cmd, [], {encoding: 'utf8', stdio: [ 'ignore', 'pipe', 'inherit' ], shell: true})
+    psql.stdout.on('data', function (data) {
+      result += data.toString()
+    })
+    psql.on('close', function (code) {
```

Comment:
personally I think it would be good to fail on nonzero even if the old CLI did not